services/forms-web-app/.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
   "singleQuote": true,
   "printWidth": 100,
-  "trailingComma": "all"
+  "trailingComma": "none"
```

Comment:
How come we've changed this?

Reply:
I was having issues with this line:

```
app.use(
  '/assets',
  express.static(path.join(__dirname, '../node_modules/govuk-frontend/govuk/assets'))
);
```

Reply:
Oh right, that would have been adding a trailing to the function - can you try `trailingComma: "es5"` and see if that works https://prettier.io/docs/en/options.html#trailing-commas

I'd prefer to keep it in so it's in the AirBnB style guide

Reply:
lovely sir! done! 👍